tests/test_units.py

```diff
@@ -17,15 +17,15 @@ def assert_converted_units(df, current, to, exp, **kwargs):
     # testing for `inplace=False` - converted values and expected unit
     _df = df.convert_unit(current, to, **kwargs, inplace=False)
 
-    npt.assert_allclose(_df._data.value, exp.values, rtol=1e-4)
+    npt.assert_allclose(_df._data.values, exp.values, rtol=1e-4)
     # When *to* is only a species symbol (e.g. 'co2e'), units are added and a
     # non-aliased symbol is returned (e.g. 'Mt CO2e'). Compare using 'in' and
     # lower().
-    assert to.lower() in _df.data.unit[5].lower()
+    assert to.lower() in _df._data.reset_index().unit[5].lower()
```

Comment:
it would be more efficient to use `get_index_levels` rather than resetting the index and selecting the columns